src/auth/__tests__/index.spec.js

```diff
@@ -133,35 +138,143 @@ describe('auth', () => {
         verifier: 'a verifier',
         redirectUri,
         state,
-        scope: 'openid'
+        scope: 'openid',
       };
       await expect(auth.exchange(parameters)).rejects.toMatchSnapshot();
     });
 
     it('should handle unexpected error', async () => {
       fetchMock.postOnce(
         'https://samples.auth0.com/oauth/token',
-        unexpectedError
+        unexpectedError,
       );
       expect.assertions(1);
       const parameters = {
         code: 'a code',
         verifier: 'a verifier',
         redirectUri,
         state,
-        scope: 'openid'
+        scope: 'openid',
       };
       await expect(auth.exchange(parameters)).rejects.toMatchSnapshot();
     });
   });
 
+  describe('passwordless flow', () => {
+    describe('with email connection', () => {
```

Comment:
Shouldn't this be `context`?

Reply:
I don't see it being used anywhere else. It could be true for mocha tests but I don't think it's worth refactoring every test file now.